src/runtime/Runtime.test.ts

```diff
@@ -1,53 +1,364 @@
-import { assert } from "console";
-import { RuntimeClass } from "./Runtime";
+export {};
 
-jest.mock("../lib/getAuth", () => {
-  return {
-    getAuth: () => {
-      return {
-        token: "testtoken",
-      };
-    },
-  };
+beforeEach(() => {
+  jest.resetModules();
+  jest.useFakeTimers();
+  jest.mock("../lib/getAuth", () => {
+    return {
+      getAuth: () => {
+        return {
+          token: "testtoken",
+        };
+      },
+    };
+  });
+  jest.mock("../lib/getProjectID", () => {
+    return {
+      getProjectID: () => {
+        return "testproject";
+      },
+    };
+  });
+  jest.mock("../api/getEvents", () => {
+    return {
+      getEvents: () => {
+        const events = require("./Runtime.events.test.json");
+        return Promise.resolve(events);
+      },
+    };
+  });
+  jest.mock("../api/getIDPool", () => {
+    return {
+      getIDPool: () => {
+        const idpool = require("./Runtime.idpool.test.json");
+        return Promise.resolve(idpool);
+      },
+    };
+  });
+  jest.mock("../api/postEvent", () => {
+    return {
+      postEvent: () => {
+        console.log("posting mock event");
+      },
+    };
+  });
 });
 
-jest.mock("../lib/getProjectID", () => {
-  return {
-    getProjectID: () => {
-      return "testproject";
-    },
-  };
+test("onReady gets called", () => {
+  jest.mock("../lib/getAuth", () => {
```

Comment:
This mock is not needed. I put this mock just to give an example of how to mock a module again. Please remove this mock.